v20.2/disaster-recovery.md

```diff
@@ -0,0 +1,365 @@
+      </td>
+    </tr>
+    <tr>
+      <td style="color:#46a417"><b>1 AZ</b></td>
+      <td style="color:#228B22"><b>√</b></td>
+    </tr>
+    <tr>
+      <td style="color:#46a417"><b>1 Region</b></td>
+      <td style="color:#228B22"><b>√</b></td>
+      <td>Check the <a href="admin-ui-overview-dashboard.html">Overview dashboard</a> on the Admin UI. If nodes are marked <b>Dead</b>, <a href="remove-nodes.html">decommission the nodes</a> and <a href="start-a-node.html">add 3 new nodes</a> in a new region. Ensure that <a href="cockroach-start.html#locality">locality flags are set</a> correctly upon node startup.</td>
+    </tr>
+    <tr>
+      <td style="color:#46a417"><b>2 or More Regions</b></td>
+      <td style="color:#FF0000"><b>X</b></td>
+      <td>Cluster will be unavailable. <br><br>Potential data loss between last backup and time of outage if region and nodes did not come back online. This would be a considerable disaster (i.e., 2 or more data centers destroyed).</td>
+      <td>When the regions come back online, try <a href="cockroach-start.html">restarting the nodes</a> in the cluster. <br><br>If the regions do not come back online and nodes are lost or destroyed, try <a href="restore.html">restoring the latest cluster backup</a> into a new cluster.<br><br>You can also <a href="https://support.cockroachlabs.com">contact Cockroach Labs support</a> for assistance.</td>
+    </tr>
+  </tbody>
+</table>
+
+<span style="color:#228B22"><b>√</b></span> = Available
+<span style="color:#FF0000"><b>X</b></span> = Outage
+
+{{site.data.alerts.callout_info}}
+When using Kubernetes, recovery actions happen automatically in many cases and no action needs to be taken.
+{{site.data.alerts.end}}
+
+## Data failure
+
+When dealing with data failure due to bad actors, rogue applications, or data corruption, domain expertise is required to identify the affected rows and determine how to to remedy the situation (e.g., remove the incorrectly inserted rows, insert deleted rows, etc.). However, there are a few actions that you can take for short-term remediation:
+
+- If you are within the garbage collection window, [run differentials](#run-differentials).
+- If you have a backup file, [restore to a point in time](#restore-to-a-point-in-time).
+- If your cluster is running and you do not have a backup with the data you need, [create a new backup](#create-a-new-backup).
+- To [recover from corrupted data in a database or table](#recover-from-corrupted-data-in-a-database-or-table), restore the corrupted object.
+
+{{site.data.alerts.callout_success}}
+To give yourself more time to recover and clean up the corrupted data, put your application in “read only” mode and only run [`AS OF SYSTEM TIME`](as-of-system-time.html) queries from the application.
+{{site.data.alerts.end}}
+
+### Run differentials
```

Comment:
So apparently, we just learned that if you DROP a database, that database cannot be referenced with AOST queries, so maybe a note... and a reference to the github issue? The workaround is taking a backup backdated to the system time where the database existed should capture it.